packages/html-reporter/mockups/public/jscpd-report.json

```diff
@@ -4924,7 +4924,7 @@
 tasks.\n   * @see DAOService\n   * @see IModelSearch\n   */\n  public static IPersistenceService getPersistenceService() {\n    Assert.isTrue(InternalOwl.getDefault().isStarted(), \"The Owl facade has not yet finished initialization\"); //$NON-NLS-1$\n    return InternalOwl.getDefault().getPersistenceService();\n  }\n\n  /**\n   * Provides access to the connection service of RSSOwl. This service provides\n   * API to load data from the internet (e.g. loading the contents of a feed).\n   * It is also the central place to ask for credentials if a resource requires\n   * authentication. Several extension points allow to customize the behavior of\n   * this service, including the ability to register\n   * <code>IProtocolHandler</code> to define the lookup process on per protocol\n   * basis or contributing <code>ICredentialsProvider</code> to define how\n   * credentials should be stored and retrieved.\n   *\n   * @return Returns the service responsible for all connection related tasks.\n   * @see IProtocolHandler\n   * @see ICredentialsProvider\n   */\n  public static IConnectionService getConnectionService() {\n    Assert.isTrue(InternalOwl.getDefault().isStarted(), \"The Owl facade has not yet finished initialization\"); //$NON-NLS-1$\n    return InternalOwl.getDefault().getConnectionService();\n  }\n\n  /**\n   * Provides access to the interpreter service of RSSOwl. This service provides\n   * API to convert a stream of data into a model representation. In the common\n   * case of a XML stream this involves using a XML-Parser and creating the\n   * model out of the content. Various extension points allow to customize the\n   * behavior of the interpreter:\n   * <ul>\n   * <li>Contribute a new format interpreter using the FormatInterpreter\n   * extension point. This allows to display any XML in RSSOwl as Feed.</li>\n   * <li>Contribute a new namespace handler using the NamespaceHandler extension\n   * point. This allows to properly handle any new namespace in RSSOwl.</li>\n   * <li>Contribute a new element handler using the ElementHandler extension\n   * point. This makes RSSOwl understand new elements or even attributes.</li>\n   * <li>Contribute a new xml parser using the XMLParser extension point if you\n   * are not happy with the default one.</li>\n   * </ul>\n   *\n   * @return Returns the service responsible for interpreting a resource.\n   * @see IFormatInterpreter\n   * @see IElementHandler\n   * @see INamespaceHandler\n   * @see IXMLParser\n   */\n  public static IInterpreterService getInterpreter() {\n    Assert.isTrue(InternalOwl.getDefault().isStarted(), \"The Owl facade has not yet finished initialization\"); //$NON-NLS-1$\n    return InternalOwl.getDefault().getInterpreter();\n  }\n\n  /**\n   * Triggers the startup sequence of the Owl core. Will return immediately if\n   * the core has already been started.\n   *\n   * @param monitor A progress monitor to report progress on long running\n   * operations (e.g. migration).\n   */",
```

Comment:
It's possible (likely?) that these are from third party libraries. If someone points them out to me, I can send a PR to them (It seems reasonable to drop this change, although it's just a mockup, so it doesn't really matter if the source is correct...):
```diff
-   * authentication. Several extension points allow to customize the behavor of
+   * authentication. Several extension points allow to customize the behavior of
```

```diff
-   * behavor of the interpreter:
+   * behavior of the interpreter:
```